accounting/src/creditcommons/routes.ts

```diff
@@ -32,11 +29,12 @@ function setResponseTrace(req: Request, res: Response) {
  */
 export function getRoutes(controller: SharedController) {
   const router = Router()
+  router.use(ccErrorHandler)
```

Comment:
I think the error handler must be registered after the routes.

Reply:
OK, let me add a unit test for it actually

Reply:
Indeed, you were right. Sorry, I should have added that unit test yesterday before re-requesting review. Fixed now.